### DISCUSSION TITLE
Add function to swap hands / mirror layout

### DISCUSSION THREAD

**Discussion by ghost** [monkeytypegame/monkeytype#2760] [Ideas]
Hi,

I'm left handed... Moreover the qwerty layout is left heavy.

It would be nice to select a swap hands button (for behaviour and appearence). This would mirror the keys in the simulator, so that right hand heavy layout can easily be tested but with a left heavy hand.

**Comment by Beethoven-n** [comment]
people have suggested that i do mirror nerps after i said that i preferred vowels on the left hand! it'd be nice to be able to do this

**Reply by Beethoven-n** [reply]
there's now an issue about it!
https://github.com/monkeytypegame/monkeytype/issues/5573
